src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -108,7 +108,7 @@ public sealed class WordBagEstimator : TrainedWrapperEstimatorBase
         public override TransformWrapper Fit(IDataView input)
         {
             // Create arguments.
-            var args = new WordBagBuildingTransformer.Arguments
+            var args = new WordBagBuildingTransformer.Options
```

Comment:
>args [](start = 16, length = 4)

same here, could you rename to options?

Reply:
fixed

---
In reply to: [254075392](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254075392) [](ancestors = 254075392)